ontotext-yasgui-web-component/src/pages/view-modes/main.js

```diff
@@ -11,6 +11,11 @@ ontoElement.config = {
 
 function renderMode(mode) {
   ontoElement.config = {
```

Comment:
I've changed this in my next upcoming MR

Reply:
I need to change it because the tests are failing.